### DISCUSSION TITLE
Detalles/Duda con la práctica 1

### DISCUSSION THREAD

**Discussion by AlbertoSF99** [Launch-X-Latam/MisionFrontEnd#140] [Q&A]
Buenos días/tardes/noches a todos, me surgió una pequeña duda con respecto a los puntos que se han marcado para la realización exitosa de la práctica. El primer punto plantea lo que se requiere, es decir, automatizar las demandas de clientes en un sitio web/aplicación móvil, y menciona que se busca que el cliente rellene un formulario. Luego, en el punto 3 especifica que para que el cliente reciba seguimiento de su demanda, deberá crearse una cuenta para ello, sin embargo, mi duda es ¿si para el formulario que indica en el primer y segundo punto se debe realizar con la cuenta creada o la cuenta es para que simplemente se le de seguimiento?. A mi parecer, debería primero pedírsele al usuario/cliente que se loguee o crearse una cuenta para poder rellenar el formulario, ¿o ustedes que opinan?

**Comment by RamaAlves** [comment]
yo creo que esa es una decisión que queda abierta para que puedas tomarla vos. Podes completar el formulario y con los datos que cargas en el formulario ya los usas para crear el perfil y luego solo le pedís que cree una contraseña. O podes crear primero la cuenta y luego completar el formulario.

**Comment by iam-leo** [comment]
Lo que plantea esta práctica es una situación real entre un cliente y una empresa/programador, donde el cliente pide "automatizar las demandas de sus clientes".
Cada punto que tu mencionas es información que te dan, pero no necesariamente es el flujo de como funcionará la web/app. Tu deber es analizar cada punto y tomar los requerimientos necesarios, teniendo en cuenta que hay información explícita e implícita como por ej cuando dice que el usuario deberá crear una cuenta para recibir el seguimiento de su demanda, no dice nada de "iniciar sesión" pero lógicamente cuando diagrames tu flujo en algún lado tendrás que colocar el inicio de sesión. Lo mismo para el formulario tu verás que decisión tomas si crear primero la cuenta y después el formulario o viceversa.
Es mi punto de vista, pero espero que haya podido ayudarte!
